visio_ml/src/modules/compile_results.py

```diff
@@ -51,30 +55,33 @@ def get_average_range(self, side) -> float:
         high.sort(key=lambda x: -hist[x])
 
         return max(high[:min(len(high), 5)]) - min(low[:min(len(low), 5)])
+    
+    def get_timestamped_results(self):
```

Comment:
Since self.delta is fixed and time between each data point is fixed (20 FPS = 0.05s between each data point), you can do `res[key] = arr[::self.delta/0.05]`